pkg/security/resolvers/process/resolver_ebpf.go

```diff
@@ -369,7 +369,7 @@ func (p *EBPFResolver) enrichEventFromProc(entry *model.ProcessCacheEntry, proc
 	// Get the file fields of the process binary
 	info, err := p.retrieveExecFileFields(procExecPath)
 	if err != nil {
-		if !os.IsNotExist(err) && !errors.Is(err, lib.ErrKeyNotExist) {
+		if !os.IsNotExist(err) {
```

Comment:
Should we do the same for the error on this line?
https://github.com/DataDog/datadog-agent/pull/33044/files#diff-873ec299bfc33246498187f7a85bd4b5d6e862571bcc87dc8bc32088a3f83927R387

Reply:
Done